Dockerfile

```diff
@@ -25,5 +25,6 @@ RUN python3 -m pip install -r requirements.txt
 COPY . /opt/chemdash
 
 # In production we override with gunicorn, etc. 
-ENTRYPOINT /opt/chemdash/chemdash.py
+ENTRYPOINT ["/opt/chemdash/chemdash.py"]
+CMD ["--help"]
```

Comment:
This got around some quirks w/ running the container.  Container runs can specify different args to chemdash.py, and if not specified `--help` is shown.